prover/crates/bin/prover_autoscaler/src/global/scaler.rs

```diff
@@ -1618,7 +1618,7 @@ mod tests {
         };
 
         assert_eq!(
-            scaler.calculate(&"prover".into(), 2 * 1500 + 1 * 3000 - 1500, &clusters),
```

Comment:
let's revert change in this file, because they block CI from passing

Reply:
This was from the lint.